frontend/src/components/navbar.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+const Navbar = () => {
+  return (
+    <nav style={{
+      backgroundColor: '#1e293b', 
+      padding: '2rem 6rem', 
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      fontWeight: 'bold' 
+    }}>
+      <div style={{
+        color: '#ffffff', 
+        fontSize: '1.5rem',
+        fontWeight: 'bold',
+        cursor:'pointer',
+      }}>
+        Exam-crack
+      </div>
+
+      <div style={{
+        display: 'flex',
+        gap: '1.5rem', 
+        padding: '0 1rem' 
+      }}>
+        <button style={{
```

Comment:
Might as well add a new `.navbar-button` class to the CSS file and use it for the buttons in here to avoid repetition